docs/user-guide/configure-certificates.md

```diff
@@ -1,30 +1,130 @@
+Zowe uses certificates to verify the identity and subsequently establish an encrypted network connection between applications. The connection is using the Secure Sockets Layer/Transport Layer Security (SSL/TLS) protocol. Certificates and their associated private keys need to be stored either in a SAF key ring or in a `PKCS12` keystore.
 
-To disable the trust store validation of southbound certificates, set the value `zowe.verifyCertificates: DISABLED` in the `zowe.yaml` configuration file.  A scenario when this is recommended is if the z/OSMF certificate is be self-signed or uses an untrusted CA. In this case, Zowe API Mediation Layer does not recognize the signing authority.  
+## What certificates Zowe supports?
 
-To enable certificate validation without hostname validation, set `zowe.verifyCertificates: NONSTRICT`. Using this setting, the certificate Common Name or Subject Alternate Name (SAN) is not checked. This facilitates deployment to environments where certificates are valid but do not contain a valid hostname. This configuration is for development purposes only and should not be used for production.
+Zowe supports keystores and truststores that are either z/OS key rings (when on z/OS) or PKCS12 files.
 
-A proper setup of trust store is mandatory to successfully start Zowe with `zowe.verifyCertificates: STRICT`.
+### PKCS12 certificates in a keystore
 
-## Keystore versus key ring
+Zowe is able to use PKCS12 certificates that are stored in USS. This certificate is used for encrypting TLS communication between Zowe clients and Zowe z/OS servers, as well as intra z/OS Zowe server to Zowe server communication. Zowe uses a `keystore` directory to contain its external certificate, and a `truststore` directory to hold the public keys of servers which Zowe communicates with (for example z/OSMF).
 
-Zowe supports certificates that are stored either in a USS directory **Java KeyStore** `.p12` format or else held in a **z/OS Keyring**.  z/OS keystore are the preferred choice for storing certificates where system programmers are already familiar with their operation and usage.  The user ID setting up a keystore and connecting it with certificates requires elevated permissions, and in scenarios where you need to create a Zowe sandbox environment or for testing purposes and your TSO user ID doesn't have authority to manipulate key rings, USS keystores are a good alternative.  
+By default, Zowe is reading PKCS12 keystore from `keystore` directory which can be located in zowe.yaml. This directory contains a server certificate, the Zowe generated certificate authority, and a `truststore` which holds intermediate certificates of servers that Zowe communicates with (for example z/OSMF).
```

Comment:
I think this would read better: "By default, Zowe reads the PKCS12 keystore from the `keystore` directory ..."